packages/icons/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@harness/icons",
-  "version": "1.68.0",
+  "version": "1.69.0",
```

We should be updating the patch version here I believe. We are not adding any new functionality
```suggestion
  "version": "1.68.1",
```